docs/dev/jobs-config-hail.json

```diff
@@ -0,0 +1,30 @@
+{
+  "new_cluster": {
+    "spark_version": "8.2.x-scala2.12",
+    "node_type_id": "Standard_DS3_v2",
+    "num_workers": 2,
+    "spark_env_vars": {
+        "HAIL_VERSION": "0.2.65"
```

Comment:
The version on which our tests are run is Hail 0.2.58. I believe that Hail upgraded directly from 2.4.5 to 3.1.1 in 0.2.65. We should not guarantee that this will work either.

Reply:
The safest thing would be a downgrade to DBR 6.4, which is supported until December; we also have Glow releases for Spark 2. We will need to cut a new Glow release to migrate to Spark 3.1.

Reply:
OK, let's wait until the new Glow release
Reluctant to have this run on Hail with Spark 2 and old versions of hail, 
since we have had people ask for hail and glow to work together with spark 3